src/ui/components/address-action/AddressActionDetails/AddressActionDetails.tsx

```diff
@@ -32,7 +32,9 @@ export function AddressActionDetails({
   showApplicationLine: boolean;
   singleAssetElementEnd: React.ReactNode;
 }) {
-  const applicationLineVisible = showApplicationLine && addressAction?.label;
+  const applicationLineVisible =
+    showApplicationLine && addressAction?.label && !recipientAddress;
+
   return (
     <>
       {recipientAddress && addressAction?.type.value === 'send' ? (
```

Comment:
These conditions don't match, but they should, as I understand

Reply:
Or maybe this isn't incorrect. But another idea: what if we always show network icon on RecipientLine?